samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -345,6 +349,18 @@ def _clean_running_containers_and_related_resources(self) -> None:
         cast(WarmLambdaRuntime, self.lambda_runtime).clean_running_containers_and_related_resources()
         cast(RefreshableSamFunctionProvider, self._function_provider).stop_observer()
 
+    def _add_account_id_to_global(self):
+        # Attempt to get the Account ID from the current session (if any)
```

Comment:
Let's replace this comment with a doctstring

Reply:
done